common/spring-cloud-stream-modules-common-configuration/src/main/java/org/springframework/cloud/stream/module/gemfire/GemfireRegionProperties.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.gemfire;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author David Turanski
+ */
+@ConfigurationProperties
+public class GemfireRegionProperties {
+	/**
+	 * The region name.
+	 */
+	@Value("${spring.application.name:}")
```

Comment:
I'm open to suggestion. This is a common pattern in modules, i.e., use stream name as default value for something.